lib/vehicle.rb

```diff
@@ -22,4 +26,29 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def date_registered
```

Comment:
usually method names are verbs since they perform an action, so naming this `add_registered_date` would be better or something like that.